### DISCUSSION TITLE
求教，不小心用了错误的chromium路径作为环境变量，如何修改为正确的路径呢？

### DISCUSSION THREAD

**Discussion by Nigel-NI** [Arondight/Adachi-BOT#462] [Q&A] [answered]
我试过了修改~/.bashrc，再执行npm install，但是bot中的路径还是原先错误路径，请问如何正确修改？

**Comment by Arondight** [comment, marked as answer]
`source ~/.bashrc`

**Reply by Nigel-NI** [reply]
抱歉，输入这个指令后重新构建，还是失败。如果是先uninstall再重新安装，能解决问题吗？如果可以，求正确的卸载方法

**Reply by Nigel-NI** [reply]
OK了，重启了一下，正常了

**Reply by Arondight** [reply]
`rm -rf node_modules`
